test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -480,7 +480,7 @@ public async Task ListFunctions_Succeeds()
             var response = await _fixture.Host.HttpClient.SendAsync(request);
             var metadata = (await response.Content.ReadAsAsync<IEnumerable<FunctionMetadataResponse>>()).ToArray();
 
-            Assert.Equal(16, metadata.Length);
+            Assert.Equal(7, metadata.Length);
```

Comment:
@fabiocav @mathewc, this PR makes it so that "List Functions" honors functions whitelist, and doesn't return non-whitelisted functions. It does make sense. Had a brief chat with @mathewc offline.

Thoughts?

Reply:
Fabio pointed out that we do document `Functions` in host.json-- https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json#functions

So, to avoid a breaking change here, I am going to keep the behavior as is -- include all functions in list functions call. I will push an update shortly.

Reply:
I updated this to not change anymore.

It seems that "Sync Triggers" call also indeed ignores the `Functions` property. Though, with this change that "Sync Triggers" will start honoring this. And I think it makes sense. As there's no reason to monitor those functions.